internal/cmd/genconfig/genconfig.go

```diff
@@ -295,13 +295,14 @@ jobs:
       if: success()
       run: ./_scripts/runDockerRun.sh
     - name: Tidy up
+      if: success() || failure()
       run: ./_scripts/postRun.sh
     - name: Upload artefacts
-      if: success() && env.CI_SKIP_JOB != 'true'
+      if: (success() || failure()) && env.CI_UPLOAD_ARTIFACTS == 'true' && !(env.CI_SKIP_JOB == 'true')
```

Comment:
I'm not sure this is right. We only want to run this step if the previous step succeeded. Why is the previous logic here not right? Because the addition of the env var means that we will only ever run this step if the previous step ran to completion, and hence success.

Reply:
Per Slack, you are 100% right. My apologies. What's definitely changed here is something on the GitHub side.